### PR TITLE
Add option to silence logger for MAST enable_cloud_dataset()

### PR DESCRIPTION
**Pull request by pllim**
Would be nice if I can silence this without affecting the logger behavior as a whole. I don't need it to remind me about S3 pricing every time I run the function.

🙏 @ceb8  🙏

### CODE REVIEW

File: astroquery/mast/core.py [hunk 1]
```diff
@@ -1561,6 +1561,8 @@ def enable_cloud_dataset(self, provider="AWS", profile=None):
             though at the moment this argument is ignored.
         profile : str
             Profile to use to identify yourself to the cloud provider (usually in ~/.aws/config).
+        verbose : bool
+            Logger to display extra info and warning.
```
Comment (ceb8) [inline astroquery/mast/core.py]: Add "Default True."

Comment (pllim) [reply astroquery/mast/core.py]: Done.

**Comment by bsipocz** [comment]
@pllim - I'm on the verge whether this needs a changelog or not, leave it up to you, and will merge once CI is happy.

**Comment by pllim** [comment]
I cancelled the CI because I plan to add a change log, squash, and then let it run.

**Pull request merged**